docs/en/query_language/functions/hash_functions.md

```diff
@@ -305,10 +305,42 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
 └──────────────────┴─────────────────┘
 ```
 
-## xxHash32, xxHash64
+## xxHash32, xxHash64 {#hash_functions-xxhash32}
 
-Calculates xxHash from a string.
-Accepts a String-type argument. Returns UInt64 Or UInt32.
-For more information, see the link: [xxHash](http://cyan4973.github.io/xxHash/)
+Calculates `xxHash` from a string. It is proposed in two flavors, 32 and 64 bits.
 
-[Original article](https://clickhouse.yandex/docs/en/query_language/functions/hash_functions/) <!--hide-->
+```sql
+SELECT xxHash32('srting');
```

Comment:
Why do we need `srting` in this example?

Reply:
Done